bio-as-utils.scm

```diff
@@ -76,14 +76,14 @@
        (string-starts-with? (cog-name A) "SMP0")))
 
 (define (go? A)
-  (and (eq? (cog-type A) 'ConceptNode)
+  (and (or (eq? (cog-type A) 'BiologicalProcessNode) (eq? (cog-type A) 'MolecularFunctionNode) (eq? (cog-type A) 'CellularComponentNode))
        (string-starts-with? (cog-name A) "GO:")))
 
 (define (gene? A)
   (and (eq? (cog-type A) 'GeneNode)))
 
 (define (GO_term? A)
-  (and (eq? (cog-type A) 'ConceptNode)
+  (and (or (eq? (cog-type A) 'BiologicalProcessNode) (eq? (cog-type A) 'MolecularFunctionNode) (eq? (cog-type A) 'CellularComponentNode))
```

Comment:
I'm not sure that's correct since "GO_term" is the top GO concept (and maybe it should be a meta-concept, i.e. be a set of GO categories as members, as opposed to as subsets).